### ISSUE TITLE
LandingPage Missing Heading Hierarchy

### ISSUE THREAD

**Issue by johnhbenetech** [benetech/MathShare#778]
There are no accessible headings whatsoever in the content of this page.  Several < div> elements are visually styled as headings, and < hr> separators are used as section boundaries.  Neither of these are sufficient to accessibly indicate the semantic structure of the page content. 
 
When combined with the lack of ARIA landmark regions, screen reader users will be unable to navigate this page in any meaningful way.  The only option available to them is to read the page from start to finish using the arrow keys. 
 
Suggested Solutions 
 
The structure of page content needs to be reflected in the use of accessible, semantic mark-up. 
 
Heading Hierarchy 
 
First, the page must include a correct heading hierarchy.  Correct meaning that: 
 
• the main content is preceded by a level 1 heading, which is the only level 1 heading on the page; 
• any child hierarchies started outside the main content (such as in the header or footer) begin at level 2; and 
• no heading levels are skipped (for example by following a heading at level 2 with one at level 4). 
 
Headings should be implemented using the native < h1> to < h6> HTML tags, rather than using ARIA.  We would suggest the following heading structure for the Landing Page: 
 
• Level 1: “Help students show and organize their math work” 
• Level 2 (off-screen if required): “Features” 
• Level 3: “Show their work” 
• Level 3: “Accessible to ALL Learners” 
• Level 3: “LMS Integration” 
• Level 3: “Free and Open Source”

**Comment by jscholes** [comment]
@rupeshparab - this is much better, but the explanatory text, both for the page as a whole and the individual features, shouldn't be marked up as a heading.  The following can just be marked up as text:

* "Benetech Mathshare empowers students to ..."
* "Students can solve equations step-by-step ..."
* "Students with and without learning differences can use ..."
* "Use on your schools LMS ..."
* "Mathshare is a free, open source tool ..."

**Comment by rupeshparab** [comment]
I have changed the headings to <p>

**Comment by jscholes** [comment]
@rupeshparab - only the five pieces of text that I indicated should've had their heading mark-up removed.  The four subtitles above them should still be level 2 headings.

* "Show their work"
* "Accessible to ALL Learners"
* "LMS Integration"
* "Free and Open Source"

**Comment by rupeshparab** [comment]
I have changed these 4 subtitles back to heading

**Comment by jscholes** [comment]
Fixed in https://github.com/benetech/MathShare/commit/2e81cba20daa79e1fe5952b7d2d1c103728fc1d2.
